docs/gaia/installation.md

```diff
@@ -40,6 +40,37 @@ $ gaiad version
 $ gaiacli version
 ```
 
+### Install binary distribution via snap (Linux only)
+
+Gaia can be installed on various GNU/Linux distributions from the [Snapcraft.io](https://snapcraft.io/gaia) store:
```

Comment:
Can we explain the advantages / disadvantages of installation via Snap vs other methods?

Reply:
I will have to come up with a brief doc anyway soon